providers/go-feature-flag/src/main/java/dev/openfeature/contrib/providers/gofeatureflag/GoFeatureFlagProvider.java

```diff
@@ -256,6 +259,32 @@ private ErrorCode mapErrorCode(String errorCode) {
         }
     }
 
+    /**
+     * convertFlagMetadata is converting the flagMetadata object received from the server
+     * to an ImmutableMetadata format known by Open Feature.
+     *
+     * @param flagMetadata - metadata received from the server
+     * @return a converted metadata object.
+     */
+    private ImmutableMetadata convertFlagMetadata(Map<String, Object> flagMetadata) {
+        ImmutableMetadata.ImmutableMetadataBuilder builder = ImmutableMetadata.builder();
+        flagMetadata.forEach((k, v) -> {
```

Comment:
We're facing `NullPointerException` on introducing the flag sdk into our server. The `metadata` should be optional.

Reply:
I am seeing this as well